apps/src/templates/courseOverview/CourseScript.js

```diff
@@ -53,7 +53,14 @@ class CourseScript extends Component {
     selectedSectionId: PropTypes.string.isRequired,
     hiddenStageState: PropTypes.object.isRequired,
     hasNoSections: PropTypes.bool.isRequired,
+    toggleHiddenScript: PropTypes.func.isRequired,
   };
+
+  onClickHiddenToggle = value => {
+    const { name, selectedSectionId, id, toggleHiddenScript } = this.props;
+    toggleHiddenScript(name, selectedSectionId, id, value === 'hidden');
+  }
```

Comment:
Nit: Missing semicolon. (Not sure we're capable of linting that yet).

Reply:
Is there reason we should actually care about a semicolon here? My initial reaction on seeing this feedback was actually confusion about where you wanted a semicolon.

This seems a little silly
```js
class {
  foo(value) {
    // code here
  }
  foo = value => {
   // code here - now I need a semicolon?
  };
}
```

Though I guess really no more silly than
```js
function foo(value) {
  // code here
}
const foo = value => {
  // code here - should have a semicolon
};
```

So I guess I'm convinced that I prob should have one, even if it might be a little silly.

Reply:
My editor asks for it :stuck_out_tongue: and it sort of makes sense as an assignment expression, instead of a member function.  I don't care _that_ much about it, but I've tried to be consistent in applying them.